index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Landify Project</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header class="header">
+    <nav>
+     <ul>
+       <li>
+          <a href="http://lokum.tech" target="_blank" rel="noopener noreferrer"> <img src="./images/Darklogo.png" alt="logo of a company">
+         </a>
+         </li>
+       <li>
+         <a href="#">About</a>
+       </li>
+       <li>
+         <a href="#">Products</a>
+       </li>
+       <li>
+         <a href="#"> Pricing</a>
+       </li>
+       <li>
+         <a href="#"> Blog</a>
+       </li>
+       <li>
+         <a href="#"> Jobs</a>
+       </li>
+       <li>
+         <label for="">
+           <select aria-label="label for the select">
+              <option value="1"> More</option>
+              <option value="2"> Something</option>
+              <option value="3"> Nobody Knows</option>
+              <option value="4">Unknown</option>
+           </select>
+         </label>
+       </li>
+     </ul>
+     <ul>
+       <li>
+         <a href="#"><img src="./images/Google Play Badge.png" alt="Google Badge">
+         </a>
+       </li>
+       <li>
+         <a href="#">
+            <img src="./images/App Store Badge.png" alt="Apple store Badge">
+         </a>
+        </li>
+     </ul>
+    </nav>
```

Comment:
Properly structure this
```
<nav>...</nav>
<section class="hero-section">...</section>
```

- Give meaningful class names to your elements. the names should kinda describe the elements

Reply:
yooo, bro you know how challenging it is. especially if you are starter.